planner/core/explain.go

```diff
@@ -306,8 +306,8 @@ func (p *PhysicalWindow) formatFrameBound(buffer *bytes.Buffer, bound *FrameBoun
 	}
 	if bound.UnBounded {
 		buffer.WriteString("unbounded")
-	} else if bound.CalcFunc != nil {
-		sf := bound.CalcFunc.(*expression.ScalarFunction)
+	} else if len(bound.CalcFuncs) > 0 {
+		sf := bound.CalcFuncs[0].(*expression.ScalarFunction)
```

Comment:
should we iterate all the `CalcFuncs`?

Reply:
No, the only case which we have multiple funcs is for `current row`, and it has already been handled.